sbin/firehol

```diff
@@ -25,31 +25,28 @@
 #       See the file COPYING for details.
 #
 
-VERSION='$Id$'
-PROGRAM_FILE="${0}"
-PROGRAM_DIR="${0%/*}"
-if [ "$PROGRAM_DIR" = "$0" ]; then PROGRAM_DIR="."; fi
+PROGRAM_FILE="$(/bin/readlink $0)"
+PROGRAM_FILE="${PROGRAM_FILE:-$0}"
```

Comment:
Guess it depends on what options busybox is compiled with: https://git.busybox.net/busybox/tree/coreutils/readlink.c seems to imply it is optional.

So I need to keep some logic with no flags to readlink for my systems but I will try to reduce the code down to make it shorter and will just use plain `readlink` with no path.